parser/sql/statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/segment/generic/WindowSegment.java

```diff
@@ -31,4 +35,15 @@ public final class WindowSegment implements SQLSegment {
     private final int startIndex;
     
     private final int stopIndex;
+    
+    private IdentifierValue identifierValue;
+    
+    private Collection<ExpressionSegment> segments;
+    
+    public WindowSegment(final int startIndex, final int stopIndex, final IdentifierValue identifierValue, final Collection<ExpressionSegment> segments) {
+        this.startIndex = startIndex;
+        this.stopIndex = stopIndex;
+        this.identifierValue = identifierValue;
+        this.segments = segments;
```

Comment:
Can you rename it to more meaningful name?

Reply:
done